src/utils.js

```diff
@@ -1088,10 +1088,16 @@ function rewriteActionsWithAdobeAuthAnnotation (packages, deploymentPackages) {
         const isWeb = checkWebFlags(thisAction.web)['web-export']
         const isRaw = checkWebFlags(thisAction.web)['raw-http'] || checkWebFlags(thisAction['web-export'])['raw-http']
 
-        // check if the annotation is defined AND the action is a web action
-        if ((isWeb || isWebExport) && thisAction.annotations && thisAction.annotations[ADOBE_AUTH_ANNOTATION]) {
+        // check if the annotation is defined
+        if (thisAction.annotations && thisAction.annotations[ADOBE_AUTH_ANNOTATION]) {
```

Comment:
small nit: we can use optional chaining now. `thisAction.annotations?.[ADOBE_AUTH_ANNOTATION]`

Reply:
Nice catch, let me update the PR